x/leverage/types/expected_types.go

```diff
@@ -2,8 +2,14 @@ package types
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 )
 
+// AccountKeeper defines the expected account keeper used for leverage simulations (noalias)
+type AccountKeeper interface {
```

Comment:
@RafilxTenfen @toteki do we actually need this or is this left over code from previous attempts?